src/Inertia.php

```diff
@@ -12,15 +12,13 @@
  * @method static void version(\Closure|string|null $version)
  * @method static string getVersion()
  * @method static \Inertia\LazyProp lazy(callable $callback)
+ * @method static \Inertia\AlwaysProp always(callable $callback)
```

Comment:
Needs to be updated for new supported types.

```suggestion
 * @method static \Inertia\AlwaysProp always(mixed $value)
```